client/src/views/Profile/Card/cardFooter.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="profile-footer">
+    <nav class="stats row">
+      <router-link to="/followers">
+        <a href="#" class="stat">
+          <div class="stat-count" v-if="footerCountInfo">
+            {{ footerCountInfo.followersCount }}
+          </div>
+          <div class="stat-titile">
+            Followers
+          </div>
+        </a>
+      </router-link>
+      <router-link to="/posts">
+        <a href="#" class="stat">
+          <div class="stat-count" v-if="footerCountInfo">
+            {{ footerCountInfo.postsCount }}
+          </div>
+          <div class="stat-titile">
+            Posts
+          </div>
+        </a>
+      </router-link>
+      <router-link to="/following">
+        <a href="#" class="stat">
+          <div class="stat-count" v-if="footerCountInfo">
+            {{ footerCountInfo.followingCount }}
+          </div>
+          <div class="stat-titile">
+            Following
+          </div>
+        </a>
+      </router-link>
+    </nav>
+    <div class="social-accounts">
+      <a v-for="account in socialAccounts" :title="`Come and find me in the ${account.type}`" :key="account.type" :href="account.link" class="social-button" :class="`social-button--${account.type}`">
+        <i v-if="account.type == 'facebook'" class="fab fa-facebook-f"></i>
+        <i v-else-if="account.type == 'youtube'" class="fab fa-youtube"></i>
+        <i v-else-if="account.type == 'instagram'" class="fab fa-instagram"></i>
```

Comment:
Could you simplify logic with dynamic classes, for example fab fa-${account.type}